ads/google/a4a/performance.js

```diff
@@ -188,42 +158,59 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
   }
 
   /**
+   * The special variable SLOT_ID will be substituted into either parameter
+   * names or values with the ID of the ad slot on the page.
+   *
    * @param {string} name  Stage name to ping out.  Should be one of the ones
    * from `LIFECYCLE_STAGES`.  If it's an unknown name, it will still be pinged,
    * but the stage ID will be set to `9999`.
    * @override
    */
   sendPing(name) {
-    this.emitPing_(this.buildPingAddress_(name));
+    const url = this.buildPingAddress_(name, this.extraVariables_);
+    if (url) {
```

Comment:
Sorry if I already asked this but in what cases would url not be populated here?